### PR TITLE
Fix popup not init in EDT for remote

### PR DESCRIPTION
**Pull request by andrewyuq**
1. BackendComponentBePopupBuilder.createPopup somehow needs to be executed in EDT while recent changes made our initPopup() to happen in BGT.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

## Checklist
- [ ] My code follows the code style of this project
- [ ] I have added tests to cover my changes
- [ ] A short description of the change has been added to the **[CHANGELOG](https://github.com/aws/aws-toolkit-jetbrains/blob/master/CONTRIBUTING.md#contributing-via-pull-requests)** if the change is customer-facing in the IDE.
- [ ] I have added metrics for my changes (if required)
 
## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: plugins/amazonq/codewhisperer/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codewhisperer/service/CodeWhispererService.kt [hunk 1]
```diff
@@ -206,9 +207,12 @@ class CodeWhispererService(private val cs: CoroutineScope) : Disposable {
...
 
-    internal fun invokeCodeWhispererInBackground(requestContext: RequestContext): Job {
-        val popup = CodeWhispererPopupManager.getInstance().initPopup()
-        Disposer.register(popup) { CodeWhispererInvocationStatus.getInstance().finishInvocation() }
+    internal suspend fun invokeCodeWhispererInBackground(requestContext: RequestContext): Job {
+        val popup = withContext(EDT) {
+            val it = CodeWhispererPopupManager.getInstance().initPopup()
+            Disposer.register(it) { CodeWhispererInvocationStatus.getInstance().finishInvocation() }
+            it
+        }
```
Comment (rli) [inline plugins/amazonq/codewhisperer/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codewhisperer/service/CodeWhispererService.kt:215]:
```suggestion
        val popup = withContext(EDT) {
            CodeWhispererPopupManager.getInstance().initPopup().also {
                Disposer.register(it) { CodeWhispererInvocationStatus.getInstance().finishInvocation() }
            }
        }
```

**Review by rli — APPROVED** [review]
changelog?

**Review by Will-ShaoHua — APPROVED** [review]

**Pull request merged**